modelservice/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2018 Expedia Group, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# Root logger option
+log4j.rootLogger=INFO, stdout
+
+# Packages and classes
+log4j.logger.com.expedia.adaptivealerting.modelservice=TRACE, stdout
```

Comment:
This will double-log. Either remove stdout from this line or else set additivity to false.
https://stackoverflow.com/questions/5699309/log4j-logging-twice